{"uid":"c1c71de239a09bee","name":"test_add_home","fullName":"case.huaxin.test_aasmoke.TestSmoke#test_add_home","historyId":"5ea5430be15809c04c2e4fa61a4ef5c9","time":{"start":1665455111435,"stop":1665455121958,"duration":10523},"description":"  测试步骤二：从首页点击进入家庭页，执行添加家庭操作，判断家庭的总数是否增加 ","descriptionHtml":"<p>测试步骤二：从首页点击进入家庭页，执行添加家庭操作，判断家庭的总数是否增加</p>\n","status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":4,"retriesStatusChange":true,"beforeStages":[{"name":"_xunit_setup_class_fixture_TestSmoke","time":{"start":1665455053351,"stop":1665455090676,"duration":37325},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"stepsCount":0,"hasContent":false}],"afterStages":[{"name":"_xunit_setup_class_fixture_TestSmoke::0","time":{"start":1665455167287,"stop":1665455169747,"duration":2460},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"stepsCount":0,"hasContent":false}],"labels":[{"name":"severity","value":"trivial"},{"name":"story","value":"添加家庭"},{"name":"feature","value":"冒烟测试"},{"name":"epic","value":"约克智慧家"},{"name":"tag","value":"repeat(1)"},{"name":"parentSuite","value":"case.huaxin"},{"name":"suite","value":"test_aasmoke"},{"name":"subSuite","value":"TestSmoke"},{"name":"host","value":"LAPTOP-TABJSFGD"},{"name":"thread","value":"11600-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"case.huaxin.test_aasmoke"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"trivial","retries":[{"uid":"e7320162705b15a5","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up\n    at JWProxy.command$ (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\jsonwp-proxy\\proxy.js:238:13)\n    at tryCatch (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:67:40)\n    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:315:22)\n    at GeneratorFunctionPrototype.prototype.(anonymous function) [as throw] (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:100:21)\n    at GeneratorFunctionPrototype.invoke (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:136:37)\n    at run (C:\\Program Files (x86)\\Appium\\resources\\app\\webpack:\\~\\babel-polyfill\\~\\core-js\\modules\\es6.promise.js:75:1)\n    at C:\\Program Files (x86)\\Appium\\resources\\app\\webpack:\\~\\babel-polyfill\\~\\core-js\\modules\\es6.promise.js:92:1\n    at flush (C:\\Program Files (x86)\\Appium\\resources\\app\\webpack:\\~\\babel-polyfill\\~\\core-js\\modules\\_microtask.js:18:1)\n    at process._tickCallback (internal/process/next_tick.js:61:11)","time":{"start":1665455024668,"stop":1665455024678,"duration":10}},{"uid":"1400cebb07045b8f","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up\n    at JWProxy.command$ (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-base-driver\\lib\\jsonwp-proxy\\proxy.js:238:13)\n    at tryCatch (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:67:40)\n    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:315:22)\n    at GeneratorFunctionPrototype.prototype.(anonymous function) [as throw] (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:100:21)\n    at GeneratorFunctionPrototype.invoke (C:\\Program Files (x86)\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\babel-runtime\\regenerator\\runtime.js:136:37)\n    at run (C:\\Program Files (x86)\\Appium\\resources\\app\\webpack:\\~\\babel-polyfill\\~\\core-js\\modules\\es6.promise.js:75:1)\n    at C:\\Program Files (x86)\\Appium\\resources\\app\\webpack:\\~\\babel-polyfill\\~\\core-js\\modules\\es6.promise.js:92:1\n    at flush (C:\\Program Files (x86)\\Appium\\resources\\app\\webpack:\\~\\babel-polyfill\\~\\core-js\\modules\\_microtask.js:18:1)\n    at process._tickCallback (internal/process/next_tick.js:61:11)","time":{"start":1665454919180,"stop":1665454930115,"duration":10935}},{"uid":"8db6b79b2bf761d7","status":"passed","time":{"start":1665453147670,"stop":1665453157853,"duration":10183}},{"uid":"24dc0e5323713f9f","status":"passed","time":{"start":1665452901890,"stop":1665452912689,"duration":10799}}],"categories":[],"tags":["repeat(1)"]},"source":"c1c71de239a09bee.json","parameterValues":[]}